content/ja/docs/concepts/workloads/pods/podpreset.md

```diff
@@ -0,0 +1,62 @@
+---
+reviewers:
+- jessfraz
+title: Pod Preset
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+このページではPodPresetについて概観します。PodPresetとは、Podの作成時にある情報をPodに注入するためのオブジェクトです。  
+その情報はシークレット、ボリューム、ボリュームマウントや環境変数などを含むことができます。
+{{% /capture %}}
+
+
+{{% capture body %}}
+## PodPresetを理解する
+
+単一の`PodPreset`はPodの作成時に追加のランタイム要求を注入するためのAPIリソースです。
+ユーザーはPodPresetを適用する対象のPodを指定するために、[ラベルセレクター](/docs/concepts/overview/working-with-objects/labels/#label-selectors)を使用します。
+
+PodPresetの使用により、Podテンプレートの作者はPodにおいて、全ての情報を明示的に指定する必要がなくなります。  
+この方法により、特定のServiceを使っているPodテンプレートの作者は、そのServiceについて全ての詳細を知る必要がなくなります。
+
+PodPresetの内部についてのさらなる情報は、[PodPresetのデザインプロポーザル](https://git.k8s.io/community/contributors/design-proposals/service-catalog/pod-preset.md)を参照してください。
+
+## PodPresetはどのように動くか
+
+Kubernetesは`PodPreset`に対する管理用コントローラーを提供し、これが有効になっている時、コントローラーはリクエストされたPod作成要求に対してPodPresetを適用します。  
+Pod作成要求が発生した時、Kubernetesシステムは下記の処理を行います。
+
+1. 使用可能な全ての`PodPreset`を取得する。
+1. それらの`PodPreset`のラベルセレクターが、作成されたPod上のラベルと一致するかチェックする。
+1. `PodPreset`によって定義された様々なリソースを、作成されたPodにマージしようと試みる。
+1. エラーが起きた時、そのPod上でマージエラーが起きたことを説明するイベントをスローし、`PodPreset`からリソースを1つも注入されていないPodを作成します。
+1. `PodPreset`によって修正されたことを示すために、マージ後の修正されたPodにアノテーションをつけます。そのアノテーションは`podpreset.admission.kubernetes.io/podpreset-<pod-preset name>: "<resource version>"`という形式になります。
```

Comment:
Should “resource version” remain in English or be translated?

Reply:
Exactly. this should be translated in Japanese. I'll fix it.